imports/api/projects.js

```diff
@@ -1,10 +1,18 @@
-import { Mongo } from "meteor/mongo";
- 
-export const Projects = new Mongo.Collection("Projects");
-
-
+import { Mongo } from "meteor/mongo"; 
 
+export const Projects = new Mongo.Collection("Projects");
 
+var projectDetails = {
```

Comment:
Is this supposed to be here or is it a leftover test that should be removed?

Reply:
We have for a person, multiple projects which can have multiple sarif files as well, so it was an implementation of the apis for both of them.

Reply:
You can check the database utilities folder where I created database sample but in SarifFiles collection we have the details for a particular testing tool report , since a person could have multiple sarif files , I have created sarifFileId and sarifFileName as a foreignKey and created an api for inserting project details which can then have relation to SarifFiles if accessed via UI in future purposes